sklearn/linear_model/tests/test_logistic.py

```diff
@@ -70,38 +69,6 @@ def test_predict_2_classes():
     check_predictions(LogisticRegression(fit_intercept=False, random_state=0), X_sp, Y1)
 
 
-def test_error():
-    # Test for appropriate exception on errors
-    msg = "Penalty term must be positive"
-
-    with pytest.raises(ValueError, match=msg):
```

Comment:
The correct way is to remove these tests. The goal of the new validation mechanism to have a common test that checks all bad values for all parameters for all estimators instead of having it on a case by case basis spread everywhere.

Reply:
Thanks for the clarification!

Reply:
Thanks for the clarification @jeremiedbb !